src/SWP/Bundle/CoreBundle/Migrations/2019/11/Version20191113081727.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SWP\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20191113081727 extends AbstractMigration
```

Comment:
the migration has some extra not needed sql queries

Reply:
which ones?

Reply:
all of them besides the ones related to `biography` field